docs/ldml/tr35-keyboards.md

```diff
@@ -515,19 +518,23 @@ _Attribute:_ `normalization`
 _Attribute:_ `layout`
 
 > The `layout` attribute describes the layout pattern, such as QWERTY, DVORAK, INSCRIPT, etc. typically used to distinguish various layouts for the same language.
+>
+> This attribute is not localized, but is an informative identifier for implementation use.
 
 _Attribute:_ `indicator`
 
-> The `indicator` attribute describes a short string to be used in currently selected layout indicator, such as US, SI9 etc.
+> The `indicator` attribute describes a short string to be used in currently selected layout indicator, such as `US`, `SI9` etc.
 > Typically, this is shown on a UI element that allows switching keyboard layouts and/or input languages.
+>
+> This attribute is not localized.
 
 * * *
 
 ### <a name="Element_names" href="#Element_names">Element: names</a>
 
 Element used to store any names given to the layout.
 
-These names are not currently localized.
+These names are not localized but are informative names for the keyboard. Localization of these names would be done as separate data items elsewhere in CLDR.
```

Comment:
should be "unique identifiers for the keyboard". That is, they should be *keys* for localized names.

Reply:
These keyboards are independently produced by their own authors and don't necessarily end up in the CLDR repo. How would you propose to allocate and enforce "unique identifiers"?  

I think this could be a good discussion, but it seems like a new item - could you file a new ticket on unique identifiers? 

The locale identifiers are not required to be unique.